### ISSUE TITLE
Vsco.co not working

### ISSUE THREAD

**Issue by thinkpad4** [RipMeApp/ripme#1526]
* Ripme version: 1.7.90
* Java version: Java 8 - 231
* Operating system: Win7 64bit

* Exact URL you were trying to rip when the problem occurred: https://vsco.co/rperron/images  But happening on all Vsco.co links
* Please include any additional information about how to reproduce the problem:
When I try to rip a Vsco.co link it shows Vsco.co album detected but when I click rip it says Expected a URL to a single image or to a member profile, got https://vsco.co/rperron/images instead
## Expected Behavior
To rip vsco.co albums
Detail the expected behavior here.

## Actual Behavior
When I try to rip a Vsco.co link it shows Vsco.co album detected but when I click rip it says Expected a URL to a single image or to a member profile, got https://vsco.co/rperron/images instead
Detail the actual (incorrect) behavior here. You can post log snippets or attach log files to your issue report.
No info in the log at all

**Comment by elontusk2232** [comment]
same error here

**Comment by chemistx** [comment]
put /1 after images,so it's going to be https://vsco.co/rperron/images/1

**Comment by thinkpad4** [comment]
@cyian-1756 can the /1 be defaulted into the code so we don't have to add the /1 every time we want to rip a Vsco album?

**Comment by cyian-1756** [comment]
@thinkpad4 

Yup, I'm not sure why it's not already. I'll get on it for the next release

**Comment by thinkpad4** [comment]
@cyian-1756  it uses Gallery now not Images https://vsco.co/rperron/gallery

**Comment by thinkpad4** [comment]
@cyian-1756 Did you add this into the latest release?

**Comment by cyian-1756** [comment]
@thinkpad4 I have not

**Comment by thinkpad4** [comment]
@cyian-1756 Any update on this?
